README.md

```diff
@@ -1,49 +1,9 @@
 # Important recent update notes - temporary note
-There are many participant fields that have been renamed to be more consistent with the rest of the package. The following fields have been renamed:
-### EL Flags
-```
-el_client_type -> el_type
-el_client_image -> el_image
-el_client_log_level -> el_log_level
-el_client_volume_size -> el_volume_size
-```
-### CL Flags
-```
-cl_client_type -> cl_type
-cl_client_image -> cl_image
-cl_client_volume_size -> cl_volume_size
-cl_client_log_level -> cl_log_level
-beacon_extra_params -> cl_extra_params
-beacon_extra_labels -> cl_extra_labels
-bn_min_cpu -> cl_min_cpu
-bn_max_cpu -> cl_max_cpu
-bn_min_mem -> cl_min_mem
-bn_max_mem -> cl_max_mem
-use_separate_validator_client -> use_separate_vc
-```
-### Validator flags
-```
-validator_client_type -> vc_type
-validator_tolerations -> vc_tolerations
-validator_client_image -> vc_image
-validator_extra_params -> vc_extra_params
-validator_extra_labels -> vc_extra_labels
-v_min_cpu -> vc_min_cpu
-v_max_cpu -> vc_max_cpu
-v_min_mem -> vc_min_mem
-v_max_mem -> vc_max_mem
-```
-### Global flags
-```
-global_client_log_level -> global_log_level
-mev_type: full -> mev_type: flashbots # new rename as of 3 May 2024
-```
+The `ethereum-package` has been moved to the [ethpandaops organization](https://github.com/ethpandaops/).
```

Comment:
at this point people will only see it on ethpandaops

GitHub will setup re directs; but this is okay

Reply:
its fine,